lib/KImg.vue

```diff
@@ -152,6 +153,13 @@
           }
         }
       },
+      onError(event) {
+        /**
+         * Emitted when the image fails to load
+         * @payload {Event} event The DOM event that triggered the error
```

Comment:
One last little note - in general it's really good to use JSDoc syntax, however in this case our documentation auto-generation doesn't handle it:

![Screenshot from 2023-12-15 19-10-50](https://github.com/learningequality/kolibri-design-system/assets/13509191/ba914720-23fa-4789-8e15-5d2887309407)

(from http://localhost:4000/kimg#events)

So I think it'd be best to just communicate this as a plain text to ensure it gets propagated to the docs. Something along the lines of

```
/**
 * Emitted when the image fails to load. The DOM event that triggered the error is available in the payload.
 * /
 ```

Reply:
Ahh okay! Didn't know about this limitation of Kolibri. Pushing the changes in a minute!